web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -142,6 +142,8 @@ object LiftRules extends LiftRulesMocker {
    */
   type StatelessTestPF = PartialFunction[List[String], Boolean]
 
+  type StackTrace = Array[StackTraceElement]
```

Comment:
Really? I dig simple type aliases like this, but please do inform what you see could be troublesome here.